dot/network/service.go

```diff
@@ -97,6 +106,7 @@ func NewService(cfg *Config, msgSend chan<- Message, msgRec <-chan Message) (*Se
 		noBootstrap:  cfg.NoBootstrap,
 		noMDNS:       cfg.NoMDNS,
 		noStatus:     cfg.NoStatus,
+		syncChan:     cfg.SyncChan,
```

Comment:
does this need to be appended here if we can already use `syncer.synChan` ?

Reply:
hmm.. yeah I think I can remove it